README.md

```diff
@@ -1,231 +1,49 @@
 
-[![npm](https://img.shields.io/npm/v/@constructor-io/constructorio-ui-autocomplete)](https://www.npmjs.com/package/@constructor-io/constructorio-ui-autocomplete)
-[![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/Constructor-io/constructorio-ui-autocomplete/blob/main/LICENSE)
+  <p align="center" style="font-size: 1.2rem;">Lightweight, minimalistic, and fully customizable autocomplete component for fast, accessible, and flexible search experiences for <a href='https://constructor.com/solutions/search'>Constructor.io's autosuggest services</a>. 🚀</p>
 
-## Introduction
+  [**Read The Docs**](https://constructor-io.github.io/constructorio-ui-autocomplete/?path=/docs/autocomplete-component--docs)
+</div>
 
-This UI Library provides React components that manage fetching and rendering logic for [Constructor.io's autosuggest services](https://constructor.com/solutions/search).
+<hr />
+<div align="center">
 
-[Our storybook docs](https://constructor-io.github.io/constructorio-ui-autocomplete) are the best place to explore the behavior and configuration options for this UI Library.
+![minzipped size](https://img.shields.io/bundlephobia/minzip/@constructor-io/constructorio-ui-autocomplete?color=green&style=flat-square)
+[![NPM Version](https://img.shields.io/npm/v/@constructor-io/constructorio-ui-autocomplete?style=flat-square)](https://www.npmjs.com/package/@constructor-io/constructorio-ui-autocomplete)
+[![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](https://github.com/Constructor-io/constructorio-ui-autocomplete/blob/main/LICENSE)
 
-![Autocomplete UI demonstration](assets/autocomplete-ui-demonstration.gif)
+<img src="assets/autocomplete-ui-demonstration.gif" alt="Autocomplete UI demonstration" height="500">
 
-## Installation
+</div>
 
-```bash
-npm i @constructor-io/constructorio-ui-autocomplete
-```
+## Features
 
-## Usage
+- 📦 A Plug and play React component
+- ⚡ Lightweight & Fast – Tiny bundle size, optimized for speed.
+- 🎨 Customizable UI – With minmal styles, and supports for custom markup.
+- ⌨️ Keyboard Navigation – Fully supports accessible keyboard naviagation.
+- ♿ Accessible (a11y) – Built-in ARIA support for screen readers
+- 🛡 Written in TypeScript with type safety.
 
-### Using the React Component
-
-The `CioAutocomplete` component handles state management, data fetching, and rendering logic.
+## Installation & Quick Start
+Install the library
+```sh
+npm i @constructor-io/constructorio-ui-autocomplete
+```
 
-```jsx
+Import and use the `CioAutocomplete` component
+```sh
 import { CioAutocomplete } from '@constructor-io/constructorio-ui-autocomplete';
-
 function YourComponent() {
   return (
     <div>
-      <CioAutocomplete apiKey="key_M57QS8SMPdLdLx4x" onSubmit={(e) => {console.log(e)}} />
+      <CioAutocomplete 
+        apiKey="key_M57QS8SMPdLdLx4x"
+        onSubmit={(e) => {console.log(e)}}
+      />
     </div>
   );
-```
```

Comment:
If we're surfacing one example, let's also include importing the styles so they get the most impact when they copy-paste the code

Reply:
Great callout